setup.py

```diff
@@ -136,29 +136,16 @@ def run(self):
               "That indicates this copy of the source code is incomplete.")
         sys.exit(2)
 
-# pycrypto or cryptography.   We choose a default but allow the user to
-# override it.  This translates into pip install of the sdist deciding what
-# package to install and also the runtime dependencies that pkg_resources
-# knows about
-crypto_backend = os.environ.get('ANSIBLE_CRYPTO_BACKEND', None)
-if crypto_backend:
-    if crypto_backend.strip() == 'pycrypto':
-        # Attempt to set version requirements
-        crypto_backend = 'pycrypto >= 2.6'
-
-    install_requirements = [r for r in install_requirements if not (r.lower().startswith('pycrypto') or r.lower().startswith('cryptography'))]
-    install_requirements.append(crypto_backend)
-
-# specify any extra requirements for installation
-extra_requirements = dict()
-extra_requirements_dir = 'packaging/requirements'
-for extra_requirements_filename in os.listdir(extra_requirements_dir):
-    filename_match = re.search(r'^requirements-(\w*).txt$', extra_requirements_filename)
-    if filename_match:
-        with open(os.path.join(extra_requirements_dir, extra_requirements_filename)) as extra_requirements_file:
-            extra_requirements[filename_match.group(1)] = extra_requirements_file.read().splitlines()
+longdesc = 'Ansible is a radically simple IT automation system. It handles configuration-management, application deployment, cloud provisioning, ad-hoc ' \
```

Comment:
I think, it's nicer to go for multiline `"""` or use parentheses for combining multiple strings instead of escaping line breaks, like:
```python
longdesc = (
    'Ansible is a radically simple IT automation system. It handles configuration-management, application deployment, cloud provisioning, ad-hoc '
    'task-execution, and multinode orchestration - including trivializing things like zero downtime rolling updates with load balancers.\n'
    '\n'
    'Read the documentation and more at https://ansible.com/'
)
```
or
```python
import textwrap
longdesc = textwrap.dedent("""
    Ansible is a radically simple IT automation system. It handles configuration-management, application deployment, cloud provisioning, ad-hoc task-execution, and multinode orchestration - including trivializing things like zero downtime rolling updates with load balancers.

    Read the documentation and more at https://ansible.com/
""")
```